packages/cli-platform-ios/src/commands/runIOS/index.ts

```diff
@@ -209,6 +211,8 @@ async function runOnSimulator(
   } else {
     const simulators = getSimulators();
 
+    const defaultSimulator = 'iPhone 14';
```

Comment:
We can now use this variable on line 224, or calculate default simulator based on the `fallbackSimulators`, like `const defaultSimulator = fallbackSimulators[0];`